wire/core/ImageSizerEngine.php

```diff
@@ -889,6 +899,22 @@ public function setQuality($n) {
 		return $this;
 	}
 
+	/**
+	 * Set the image quality 1-100 for WebP output, where 100 is highest quality
+	 *
+	 * @param int $n
+	 *
+	 * @return $this
+	 *
+	 */
+	public function setWebpQuality($n) {
+		$n = (int) $n;
```

Comment:
Could this be simplified? `$this->webpQuality = min(max((int) $n, 1), 100);`

Reply:
That would make the code less readable, IMHO. One-liners are fun and save lines of code (if that's a priority), but they can also take a while to sink in – the benefit of simple if-else-structure is that it's instantly understandable :)

... though that being said, I would recommend one change to this method as well:

```
else if($n > 100) $n = 100;
```

Micro-optimization, but if first rule is true then the second doesn't need to be processed, and this would also make the intention more obvious (again in my opinion).

Reply:
Actually here's another minor suggestion:

```
-		$this->webpQuality = (int) $n;
+		$this->webpQuality = $n;
```

`(int)` is pointless there, since we've already converted the value to an integer a few lines above. Again very small thing, but this extraneous type conversion can make one wonder if there's actually something strange going on here.